public/scripts/slash-commands/SlashCommandParser.js

```diff
@@ -1045,7 +1045,7 @@ export class SlashCommandParser {
         /**@type {SlashCommandUnnamedArgumentAssignment}*/
         let assignment = new SlashCommandUnnamedArgumentAssignment();
         assignment.start = this.index;
-        if (!split && this.testQuotedValue()) {
+        if (!split && !rawQuotes && this.testQuotedValue()) {
```

Comment:
Are you sure the interaction of `split` and `rawQuotes` are correct this way?
I am a bit confused myself, but this doesn't look right.